CHANGELOG.md

```diff
@@ -1,1929 +1,267 @@
-  ([@rinfan](https://github.com/rinfan))
-* [`0616fd22a`](https://github.com/npm/npm/commit/0616fd22a4e4f2b2998bb70d86d269756aab64be)
-  [#19216](https://github.com/npm/npm/pull/19216)
-  Add `edit` section to description in `npm-team.md`.
-  ([@WispProxy](https://github.com/WispProxy))
-* [`c2bbaaa58`](https://github.com/npm/npm/commit/c2bbaaa582d024cc48b410757efbb81d95837d43)
-  [#19194](https://github.com/npm/npm/pull/19194)
-  Tiny style fix in `npm.md`.
-  ([@WispProxy](https://github.com/WispProxy))
-* [`dcdfdcbb0`](https://github.com/npm/npm/commit/dcdfdcbb0035ef3290bd0912f562e26f6fc4ea94)
-  [#19192](https://github.com/npm/npm/pull/19192)
-  Document `--development` flag in `npm-ls.md`.
-  ([@WispProxy](https://github.com/WispProxy))
-* [`d7ff07135`](https://github.com/npm/npm/commit/d7ff07135a685dd89c15e29d6a28fca33cf448b0)
-  [#18514](https://github.com/npm/npm/pull/18514)
-  Make it so `javascript` -> `JavaScript`. This is important.
-  ([@masonpawsey](https://github.com/masonpawsey))
-* [`7a8705113`](https://github.com/npm/npm/commit/7a870511327d31e8921d6afa845ec8065c60064b)
-  [#18407](https://github.com/npm/npm/pull/18407)
-  Clarify the mechanics of the `file` field in `package.json` a bit.
-  ([@bmacnaughton](https://github.com/bmacnaughton))
-* [`b2a1cf084`](https://github.com/npm/npm/commit/b2a1cf0844ceaeb51ed04f3ae81678527ec9ae68)
-  [#18382](https://github.com/npm/npm/pull/18382)
-  Document the [`browser`
-  field](https://github.com/defunctzombie/package-browser-field-spec) in
-  `package.json`.
-  ([@mxstbr](https://github.com/mxstbr))
-
-### MISC
-
-* [`b8a48a959`](https://github.com/npm/npm/commit/b8a48a9595b379cfc2a2c576f61062120ea0caf7)
-  [#19907](https://github.com/npm/npm/pull/19907)
-  Consolidate code for stringifying `package.json` and package locks. Also adds
-  tests have been added to test that `package[-lock].json` files are written to
-  disk with their original line endings.
-  ([@nwoltman](https://github.com/nwoltman))
-* [`b4f707d9f`](https://github.com/npm/npm/commit/b4f707d9f543f0995ed5811827a892fc8b2192b5)
-  [#19879](https://github.com/npm/npm/pull/19879)
-  Remove unused devDependency `nock` from `.gitignore`.
-  ([@watilde](https://github.com/watilde))
-* [`8150dd5f7`](https://github.com/npm/npm/commit/8150dd5f72520eb143f75e44787a5775bd8b8ebc)
-  [#16540](https://github.com/npm/npm/pull/16540)
-  Stop doing an `uninstall` when using `make clean`.
-  ([@metux](https://github.com/metux))
-
-### OTHER DEPENDENCY BUMPS
-
-* [`ab237a2a5`](https://github.com/npm/npm/commit/ab237a2a5dcf70ee490e2f0322dfedb1560251d4)
-  `init-package-json@1.10.3`
-  ([@zkat](https://github.com/zkat))
-* [`f6d668941`](https://github.com/npm/npm/commit/f6d6689414f00a67a1f34afc6791bdc7d7be4d9b)
-  `npm-lifecycle@2.0.1`
+* [`a9e722118`](https://github.com/npm/npm/commit/a9e7221181dc88e14820d0677acccf0648ac3c5a)
+  [#20256](https://github.com/npm/npm/pull/20256)
+  add support for managing npm hooks
```

Comment:
Capitalisation and `.` missing

Reply:
oh good catch, this should actually say more as this is kinda exciting.